spring-integration-core/src/main/java/org/springframework/integration/support/management/CounterFacade.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.support.management;
+
+import java.time.Duration;
+
+/**
+ * A facade abstraction over some timer that tracks {@link Duration}.
+ *
+ * @author Gary Russell
+ * @since 5.0.1
+ *
+ */
+public interface CounterFacade {
```

Comment:
Micrometer is intended to be the facade over a bunch of monitoring systems. Why have a facade of the facade?

Reply:
The problem is timing (pun); micrometer is still RC so we can't have a hard dependency on it in our 5.0.1 release (due next Wednesday).